File: go/1064.go
package main

import "fmt"

func main() {
  var p int
  var n, t float64
  for i := 0; i < 6; i++ {
    fmt.Scanf("%f", &n)
    if n > 0 {
      p += 1
      t += n
    }
  }
  fmt.Printf("%d valores positivos\n", p)
  fmt.Printf("%.1f\n", t/float64(p))
}

File: lua/1064.lua
local p = 0
local n = 0
local t = 0

for i = 1, 6 do
    n = io.read("*n")

    if n > 0 then
        p = p + 1
        t = t + n
    end
end

print(string.format("%d valores positivos", p))

if p > 0 then
    print(string.format("%.1f", t / p))
else
    print("Nenhum valor positivo foi inserido")
end
